### PR TITLE
Welcome to AdaLite modal

### PR DESCRIPTION
**Pull request by iref**
Changes login disclaimer content. It's still missing icons for recommendation sections.

### CODE REVIEW

File: app/public/css/styles.css [hunk 1]
```diff
@@ -23,108 +23,26 @@
...
+  --text-color: #20323f;
+  --text-color-gray: #606a71;
   --input-text-inactive: #999;
   --input-text-active: #333;
   --input-background: #f4f5f6;
   --gray-background: #e4e5e6;
   --border: #dadada;
   --navbar: #f4f5f6;
-  --positive: #38a14f;
+  --positive: #78d088;
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/public/css/styles.css]: Will be removed by merging PR #306

File: app/frontend/components/pages/login/aboutOverlay.js [hunk 2]
```diff
@@ -26,118 +36,112 @@ class AboutOverlayClass extends Component {
...
+              checked: dontShowAgainCheckbox,
+              onChange: this.checkboxClick,
+              class: 'checkbox',
+            }),
+            "Don't show this notice again."
+          ),
+          h(
+            'button',
+            {
+              onClick: this.toggleAboutOverlay,
```
[the remark this answers is not in the document]
Comment (dejvid15) [reply app/frontend/components/pages/login/aboutOverlay.js]: Renamed

**Comment by dejvid15** [comment]
@darjanin Implemented your review comments for `Welcome` & adjusted `Modal` props as discussed.

**Comment by darjanin** [comment]
@dejvid15 Please rebase and merge.

**Pull request merged**